force-app/test/package/classes/unit/ArgumentTest.cls

```diff
@@ -372,7 +374,8 @@ public class ArgumentTest {
         return true;
       }
 
-      if ((o == null) || !(o instanceof CustomType)) {
+      // Must do instanceof check for Boolean BEFORE CustomType or else SF will GACK on `(CustomType) o`
```

Comment:
**question:** I'm curious, what is the issue here? What GACK? Is it related to the version 61.0 ?

Reply:
The issue seems to be related to handling certain types as `Object` inputs. Not sure why it is happening honestly.